xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -0,0 +1,239 @@
+  </para>
+  <para>
+   &sle; 15.1 adds support for the &rpi; 3 Model B+, which was not fully
+   functional using &sle; 15. Apart from this, the installation process is the
+   same as it was for &sle; 12 SP3. This is documented here: 
+   <link xlink:href="https://www.suse.com/documentation/suse-best-practices/sles-for-arm-raspberry-pi/data/sles-for-arm-raspberry-pi.html"/> 
+  </para>
+  
+ </sect2>
+ <sect2>
+  <title>Installation from USB Media</title>
+  <para>
+   This method will be more familiar as it is the way &sls; is deployed
+   on other platforms. Installation can be performed from a removable USB
+   medium, such as a memory stick, onto a MicroSD card in the machine's internal
+   slot, or from a removable USB medium onto another USB medium, such as a
+   USB-connected hard disk.
+  </para>
+  <note>
+   <title>USB Bandwidth Limitations</title>
+   <para>
+    Note that the Ethernet, Wi-Fi and Bluetooth controllers on the &rpi; are all
+    connected to the device's on-board USB 2 bus. Therefore an operating system
+    running from a disk attached via USB must share the total 480 MBps bandwidth
+    of the USB controller. This will limit performance, and could significantly
+    impact network performance.
+   </para>
+  </note>
+  <para>
+   Installation on the Raspberry Pi has some aspects that are unlike other
+   platforms.
+  </para>
+  <para>
+   The &rpi; differs from more conventional machines in two important ways:
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     The &rpi; hardware does not have any built-in firmware. Its firmware is
+     loaded from a special reserved FAT16 partition on the MicroSD card every
+     time that the machine is powered on. There is also no battery-backed clock
+     or freely-modifiable "CMOS" memory to hold configuration information.
+    </para>    
+    <para>
+     This means there are no conventional settings to adjust for boot device
+     order, time and date, and so on.
+    </para>
+    <para>
+     Instead, the &rpi; has a very small amount of One-Time Programmable Memory
+     (OTP memory) which can be used to configure some settings, such as whether
+     the machine should attempt to boot from USB devices or over Ethernet. It is
+     only possible to program this by preparing a bootable MicroSD card and
+     placing certain configuration files on it. 
+    </para>
+    <para>
+     This OTP memory is described on the &rpi; Foundation web site, here:
+     <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/otpbits.mdd"/>
+    </para>
+    <note>
+     <title>Time Synchronization</title>
+     <para>
+      The lack of a Real Time Clock (RTC) also means that it is essential for
+      &rpi; devices to be configured to fetch the time from a network server by
+      &ntp;.
+    </para>
+    </note>
+   </listitem>
+   <listitem>
+    <para>
+     The primary processor on the &rpi; is the GPU, not the ARM processor. It is
```

Comment:
Arm - and I'm pretty sure there's some entity for it

Reply:
ARM is in block capitals because it is an acronym; it stands for "Advanced RISC Machine", formerly "Acorn RISC Machine".

Reply:
No. It's a trademark and it was renamed in 2017, see PR #447. Please do not regress my cleanup from PR #447 and refer to the current trademark usage guidelines: https://www.arm.com/company/policies/trademarks

Reply:
Replaced with existing entity.